lib/Gitlab/Api/Projects.php

```diff
@@ -496,11 +496,15 @@ public function removeLabel($project_id, $name)
 
     /**
      * @param int $project_id
+     * @param array $params (
+     *
+     *     @var string $namespace      The ID or path of the namespace that the project will be forked to
+     * )
      * @return mixed
      */
-    public function fork($project_id)
+    public function fork($project_id, array $params = array())
     {
-        return $this->post('projects/'.$this->encodePath($project_id).'/fork');
+        return $this->post($this->getProjectPath($project_id, 'fork'), $params);
```

Comment:
Please, use the `OptionResolver` to add input data validation.

Reply:
OK